bin/reth/src/node/events.rs

```diff
@@ -81,11 +81,11 @@ impl NodeState {
             BeaconConsensusEngineEvent::ForkchoiceUpdated(state) => {
                 info!(target: "reth::cli", ?state, "Forkchoice updated");
             }
-            BeaconConsensusEngineEvent::CanonicalBlockAdded(number, hash) => {
-                info!(target: "reth::cli", number, ?hash, "Block added to canonical chain");
+            BeaconConsensusEngineEvent::CanonicalBlockAdded(block) => {
+                info!(target: "reth::cli", ?block, "Block added to canonical chain");
```

Comment:
this is very verbose,

the trace should remain the same, only logging the number and hash